### ISSUE TITLE
Tracked app version mismatch between platforms

### ISSUE THREAD

**Issue by giuseppelt** [OneSignal/OneSignal-Cordova-SDK#454]
**Description:**

To track the app version, One Signal uses platform specific build version (which is a technical info), and not the "meta" app version available (which is the real version, also available to the user).
- on iOS, it uses buildNumber, instead of the bundleVersion 
- on Android, it uses versionCode, instead of the versionName

Actually the tracked info is platform specific and not common among platform. In addition buildNumber on iOS is the wrong data to track as version, because it is not unique, it can be reused among app "real" versions.
So there's no way to address or use a condition based on the app version, across all platforms.

**Comment by jkasten2** [comment]
@giuseppelt Thanks for the suggestions.
We are tracking the change for iOS to use `bundleVersion` instead of `buildNumber` in https://github.com/OneSignal/OneSignal-iOS-SDK/issues/303.

On Android the Google Play store does not enforce you to change `versionName` when updating your app, just `versionCode`.

You are correct that `versionCode` on Android and `bundleVersion` on iOS are two different formats. On android `versionCode` is locked to an integer where iOS is almost always a version number with a dot, example 1.1, 1.2.3, etc, but can be an integer number.

In summary, we do plan to correct iOS to send `bundleVersion` in the future. However with Android may not, we taking the above into consideration.

**Comment by Nightsd01** [comment]
Closing this issue as it is an issue we are already tracking in the iOS SDK repo. Thanks for posting!
